trk/src/Navigation/AppNavigator.js

```diff
@@ -31,6 +32,11 @@ function HomeStack() {
         component={ClimbDetailScreen}
         options={{ title: 'Climb Detail', headerBackTitle: 'Home', headerTitleAlign: 'center' }}
       />
+      <Stack.Screen
```

Comment:
I your point about the Screen being implemented in two parts of the navigator. It should just be one.. Something to figure out in a future PR